apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloServerInterceptor.java

```diff
@@ -112,7 +101,7 @@ public void interceptAsync(@Nonnull final Operation operation, @Nonnull final Ap
     this.httpCall = null;
   }
 
-  private Call httpCall(Operation operation) throws IOException {
+  Call httpCall(Operation operation) {
```

Comment:
unrelated visibility change obviates the need for a synthetic accessor method. will revert if unwanted.